packages/views/screens/managers/request/request.test.js

```diff
@@ -14,106 +13,29 @@ describe('Request', () => {
   let wrapper;
 
   const props = {
-    account: {
-      info: {
-        LSK: {
-          summary: { address: accounts.genesis.summary.address },
-        },
-        BTC: {
-          summary: { address: '16Qp9op3fTESTBTCACCOUNTv52ghRzYreUuQ' },
-        },
-      },
-    },
-    token: tokenMap.LSK.key,
+    address: accounts.genesis.summary.address,
     t: v => v,
-    history: {
-      push: jest.fn(),
-      location: {
-        pathname: 'wallet',
-        search: '?modal=request',
-      },
-    },
+  };
+  const routeConfig = {
+    pathname: 'wallet',
+    search: '?modal=request',
   };
 
   beforeEach(() => {
-    wrapper = mountWithRouter(Request, props, props.history.location);
+    wrapper = mountWithRouter(Request, props, routeConfig);
   });
 
-  describe('Amount field', () => {
```

Comment:
@reyraa why this test is removed? it isn't have relation to BTC

Reply:
A copy of this is already moved to the token module.

Reply:
A copy of this is already moved to the token module.